.github/workflows/ci.yml

```diff
@@ -65,7 +62,7 @@ jobs:
         run: |
           python setup.py install
         env:
-          BUILD_S3: ${{ matrix.with-s3 }}
+          BUILD_S3: 1
```

Comment:
Is there any situation where we do not enable `BUILD_S3`? 

It looks like only when the setup is Windows with conda, why is that?

Reply:
For binary releases, it's just because there are too many Error that Idk how to solve with AWS enabled for conda on windows.

For CI testing, you can treat it as wheels, we could enable testing with awssdk.

Generally, the reason I think it makes sense to always enable AWSSDK in our testing CI is the >95% tests are the same with or without AWSSDK enabled.